sections/exchange/FooterCard/TradeSummaryCard/FeeRateSummaryItem.tsx

```diff
@@ -1,28 +1,85 @@
 import { FC } from 'react';
 import { useTranslation } from 'react-i18next';
-
-import { formatPercent } from 'utils/formatters/number';
+import styled from 'styled-components';
+import Tippy from '@tippyjs/react';
+import { Svg } from 'react-optimized-image';
 
 import { NO_VALUE } from 'constants/placeholder';
 
+import TimerIcon from 'assets/svg/app/timer.svg';
+
+import { formatPercent } from 'utils/formatters/number';
+
 import { SummaryItem, SummaryItemValue, SummaryItemLabel } from '../common';
 import Wei from '@synthetixio/wei';
 
 type FeeRateSummaryItemProps = {
-	feeRate: Wei | null;
+	totalFeeRate: Wei | null;
+	baseFeeRate: Wei | null;
 };
 
-const FeeRateSummaryItem: FC<FeeRateSummaryItemProps> = ({ feeRate }) => {
+const FeeRateSummaryItem: FC<FeeRateSummaryItemProps> = ({ totalFeeRate, baseFeeRate }) => {
 	const { t } = useTranslation();
 
 	return (
 		<SummaryItem>
 			<SummaryItemLabel>{t('exchange.summary-info.fee')}</SummaryItemLabel>
-			<SummaryItemValue data-testid="exchange-fee-rate">
-				{feeRate != null ? formatPercent(feeRate) : NO_VALUE}
+			<SummaryItemValue>
+				<FeeRateItem data-testid="exchange-fee-rate">
+					<span>
+						{baseFeeRate != null ? formatPercent(baseFeeRate, { minDecimals: 2 }) : NO_VALUE}
+					</span>
+					{totalFeeRate != null && baseFeeRate != null ? (
+						totalFeeRate.sub(baseFeeRate).toNumber() > 0 ? (
+							<>
+								<span>+</span>
```

Comment:
Can you make this light gray like the design?

Reply:
Thanks for the reminder. I didn't notice it.